doc/usage/getting_started.md

```diff
@@ -4,23 +4,37 @@ Currently we don't publish the binary build with the release, but it is pretty s
 
 ## Usage
 
-You can follow the `help` flag on `etcdbrctl` command and its sub-commands to know the usage details. Some of the common use cases are mentioned below. Although examples below use AWS S3 as storage provider, etcd-backup-restore supports AWS, GCS, Azure, Openstack swift and Alicloud OSS object store. It also supports local disk as storage provider for development purposes, but it is not recommended to use this in a production environment.
+You can follow the `help` flag on `etcdbrctl` command and its sub-commands to know the usage details. Some of the common use cases are mentioned below. Although examples below use `AWS S3` as storage provider, etcd-backup-restore supports AWS, GCS, Azure, Openstack swift and Alicloud OSS object store. It also supports local disk as storage provider for development purposes, but it is not recommended to use this in a production environment.
 
 ### Cloud Provider Credentials
 
-The procedure to provide credentials to access the cloud provider object store varies for different providers.
+The procedure to provide credentials to access the cloud provider object store varies for different providers, there are various ways to pass credentials([described below](https://github.com/gardener/etcd-backup-restore/blob/6b119df1f07c32350c40eabce70dd061fbebe78c/doc/usage/getting_started.md#various-ways-to-pass-credentials)), you can choose either ways but we recommend you to pass credentials through a file.
 
-* For `AWS S3`, the `credentials` file has to be provided in the `~/.aws` directory.
 
-* For `GCP Containers`, the service account json file should be provided in the `~/.gcp` as a `service-account-file.json` file.
+### Various ways to pass Credentials:
```

Comment:
If files are supported, can you please describe the expected structure of the file? What do you think about if we add example files to `/example` in this repo and link them in this document?

Reply:
for each providers?

Reply:
done. PTAL

Reply:
Can you please refer to the example directory in this document?